user-docs/pupil-capture.md

```diff
@@ -188,6 +188,19 @@ This method is for special situations and far distances. Usually not required.
 1. Repeat until you have covered the subjects field of view (generally about 9 points should suffice)
 1. Press `c` on your keyboard or click the blue circular `C` button in the left hand side of the world window to stop calibration.
 
+#### Camera Intrinsics Estimation
+
+This method is used to calculate camera intrinsics and correct camera distortion. Currently implemented with [pinhole camera model](http://docs.opencv.org/2.4/doc/tutorials/calib3d/camera_calibration/camera_calibration.html). Usually not required.
```

Comment:
This calibration plugin is used to calculate camera intrinsics, which will enable one to correct cor camera distortion. Pupil Capture has built in default camera matrices for the high speed world camera and the high resolution world camera. You can re-calibrate your camera and/or calibrate a camera that is not supplied by Pupil Labs by running this calibration routine.